cif_pow.dic

```diff
@@ -9690,6 +9708,25 @@ save_PD_PREP
 ;
     _name.category_id             PD_GROUP
     _name.object_id               PD_PREP
+    _category_key.name            '_pd_prep.id'
+
+save_
+
+save_pd_prep.char_id
+
+    _definition.id                '_pd_prep.char_id'
+    _definition.update            2023-06-04
+    _description.text
+;
+    A char id, representing a material, to which this sample relates.
```

Comment:
"An identifier for the material from which this sample was prepared" I think would be more understandable.